documentation/asciidoc/topics/yaml/grafana_datasource.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: integreatly.org/v1alpha1
+kind: GrafanaDataSource
+metadata:
+  name: grafanadatasource
+spec:
+  name: sso.yaml
+  datasources:
+    - name: Prometheus
+      type: prometheus
+      access: proxy
+      # Change the url value to include the namespace of the Prometheus stack.
+      url: http://prometheus-operated.<prometheus-namespace>:9090
```

Comment:
We can also check, that `prometheus-operated.<prometheus-namespace>` is enough for cross namespace communication without `.svc` post-fix as described in some articles.

Reply:
@dmvolod I do see `.svc` in the URL from those two articles you posted in the other comment. 

@Crumby Could you please confirm from your OCP cluster if that `.svc` suffix is needed here?

Reply:
I think it is. I actually tried to create a `GrafanaDatasource` on openshifer earlier today using user workload monitoring and I had issues with 401 responses from Prometheus. So an additional config might be required by the user here. I haven't had a chance to explore further yet.

Reply:
You need to configure Bearer token (ServiceAccount with `cluster-monitoring-view`) if you want to connect to user workload monitor. User workload monitor is secured with OpenShift OAuth